packages/mobile/src/account/saga.ts

```diff
@@ -70,6 +72,20 @@ function* clearStoredAccountSaga({ account, onlyReduxState }: ClearStoredAccount
   }
 }
 
+export async function generateLinkWithPath(path: string) {
+  let bundleId = DeviceInfo.getBundleId()
+  bundleId = bundleId.replace(/\.(debug|dev)$/g, '.alfajores')
+
+  const shortUrl = await generateShortLink({
+    link: `https://valoraapp.com/${path}`,
```

Comment:
Let's not hardcode the domain and use `WEB_LINK` instead.
And when I think about it, it seems this should directly be the `celo://wallet/pay?xxx` type of link.
which is returned by `urlFromUriData`.

Reply:
Ok I was wrong about using `celo://` links here:

>The link your app will open. Specify a URL that your app can handle, typically the app's content or payload, which initiates app-specific logic (such as crediting the user with a coupon or displaying a welcome screen). This link must be a well-formatted URL, be properly URL-encoded, use either HTTP or HTTPS, and cannot be another Dynamic Link.

_From: https://firebase.google.com/docs/dynamic-links/ios/create#dynamic-link-parameters_

Reply:
The short linking doesn't work if we don't pass a link from a domain we own apparently. Also, this shortlink if opened from desktop will open the valoraapp.com webpage where we can show a download app button, but with the `celo://` it would not open anything

Reply:
Yes, it also mentions the following:
> When users open a Dynamic Link on a desktop web browser, they will load this URL (unless the ofl parameter is specified). If you don't have a web equivalent to the linked content, the URL doesn't need to point to a valid web resource. In this situation, you should set up a redirect from this URL to, for example, your home page.

In that case I propose we do something like this:

```ts
link: `${WEB_LINK}?deeplink=${encodeURIComponent(urlFromUriData)}`
```

This way we don't need to make changes to `valoraapp.com` if open in a browser.

What do you think?